azurerm/internal/services/network/registration.go

```diff
@@ -39,6 +39,7 @@ func (r Registration) SupportedDataSources() map[string]*schema.Resource {
 		"azurerm_public_ips":                                dataSourceArmPublicIPs(),
 		"azurerm_public_ip_prefix":                          dataSourceArmPublicIpPrefix(),
 		"azurerm_route_table":                               dataSourceArmRouteTable(),
+		"azurerm_service_tags":                              dataSourceServiceTags(),
```

Comment:
Can we rename it as `azurerm_network_service_tags` so as to be easier reasoned about?